go/test/endtoend/utils/utils.go

```diff
@@ -282,6 +282,28 @@ func WaitForKsError(t *testing.T, vtgateProcess cluster.VtgateProcess, ks string
 	}
 }
 
+// WaitForTableDeletions waits for a table to be deleted
+func WaitForTableDeletions(t *testing.T, vtgateProcess cluster.VtgateProcess, ks, tbl string) error {
+	timeout := time.After(60 * time.Second)
+	for {
+		select {
+		case <-timeout:
```

Comment:
This is incorrect: on every single loop iteration you allow a timeout of `60s`. What you really want is a total wait of `60s`. Suggestion: use `context.WithTimeout()`.

Reply:
Okay 👍